easybuild/easyconfigs/g/gmsh/gmsh-4.12.2-foss-2023b.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'CMakePythonPackage'
```

Sorry I misunderstood the added PYTHONPATH. This doesn't install a Python package but just a single file.

```suggestion
easyblock = 'CMakeMake'
```